mesonbuild/compilers.py

```diff
@@ -213,6 +213,43 @@ def is_library(fname):
                                                           True),
                 }
 
+gnulike_instruction_set_args = {'mmx': ['-mmmx'],
+                                'sse': ['-msse'],
+                                'sse2': ['-msse2'],
+                                'sse3': ['-msse3'],
+                                'ssse3': ['-mssse3'],
+                                'sse41': ['-msse4.1'],
+                                'sse42': ['-msse4.2'],
+                                'avx': ['-mavx'],
+                                'avx2': ['-mavx2'],
+                                'neon': ['-mfpu=neon'],
+                                }
+
+vs32_instruction_set_args = {'mmx': ['/arch:SSE'], # There does not seem to be a flag just for MMX
+                           'sse': ['/arch:SSE'],
+                           'sse2': ['/arch:SSE2'],
+                           'sse3': ['/arch:AVX'], # VS leaped from SSE2 directly to AVX.
+                           'ssse3': ['/arch:AVX'],
+                           'sse41': ['/arch:AVX'],
+                           'sse42': ['/arch:AVX'],
+                           'avx': ['/arch:AVX'],
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->